cognite/neat/graph/loaders/_rdf2asset.py

```diff
@@ -150,14 +194,45 @@ def _upload_to_cdf(
         dry_run: bool,
         read_issues: NeatIssueList,
     ) -> Iterable[UploadResult]:
-        raise NotImplementedError("Not implemented yet, this is placeholder")
+        try:
+            upserted = client.assets.upsert(items, mode="replace")
+        except CogniteAPIError as e:
+            result = UploadResult[str](name="Asset", issues=read_issues)
+            result.error_messages.append(str(e))
+            result.failed_upserted.update(item.as_id() for item in e.failed + e.unknown)
+            result.created.update(item.as_id() for item in e.successful)
+            yield result
+        else:
+            for asset in upserted:
+                result = UploadResult[str](name="asset", issues=read_issues)
+                result.created.add(cast(str, asset.external_id))
```

Comment:
What about adding a `upserted` to `UploadResult`?
```suggestion
            result.upserted.update(item.as_id() for item in e.successful)
            yield result
        else:
            for asset in upserted:
                result = UploadResult[str](name="asset", issues=read_issues)
                result.upserted.add(cast(str, asset.external_id))
```

Reply:
implemented